README.md

```diff
@@ -1,31 +1,30 @@
 Themes: running Signal Desktop without a phone on linux using X11-adjacent tretchery and signal-cli with improved semantics for solving captchas and getting QR codes
 
 The general dance goes something like this:
-```
+```sh
 # install stuff  to handle the signalcaptcha:// uri redirect
 cp signalcaptcha.desktop /usr/share/applications
 cp save_signal_captcha /opt
-echo "x-scheme-handler/signalcaptcha=signalcaptcha.desktop;" >> /usr/share/applications/mimeinfo.cache
+update-desktop-database
```

Comment:
This works but then signalcaptcha.desktop needs to be renamed to lexographically come before signal-desktop.desktop

Reply:
Fixed